pom.xml

```diff
@@ -286,12 +286,21 @@
                     <group>JBoss SET/Tools</group>
                     <packager>JBoss SET</packager>
                     <mappings>
+                        <!-- not very elegant, but only way I found to ensure the proper files
+                             attributes were set was to "override" the first mapping with a second
+                             one ... -->
                         <mapping>
                             <directory>/usr/share/${project.name}</directory>
                             <filemode>644</filemode>
-                            <username>jboss</username>
-                            <groupname>jboss</groupname>
-                            <artifact />
+                            <username>root</username>
+                            <groupname>root</groupname>
+                            <artifact/>
+                        </mapping>
+                        <mapping>
+                            <directory>/usr/share/${project.name}</directory>
```

Comment:
Yes, this is the only solution I found to have the files properly set. It's ugly, but I have not found a way to do it properly with the maven-rpm-plugin.